#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

int32 cost

int32 north
int32 east
int32 south
int32 west